vllm/model_executor/parallel_utils/pynccl.py

```diff
@@ -34,18 +35,24 @@
 
 so_file = os.environ.get("VLLM_NCCL_SO_PATH", "")
 
+# check if we have vllm-managed nccl
+cuda_major = torch.version.cuda.split(".")[0]
```

Comment:
QQ: Does this work with AMD?

Reply:
Emm, good question. Fixed in [bbf6afe](https://github.com/vllm-project/vllm/pull/3805/commits/bbf6afe2ba0e102931e44669b0764e6a9f149ca1).